buildSrc/src/main/kotlin/ReleaseScripts.kt

```diff
@@ -134,12 +134,20 @@ fun updateGeneratorVersion(gitDirectory: File, versionConfigFile: File) {
     commitChangesInFile(gitDirectory, versionConfigFile, commitMessage)
 }
 
-fun updateProviderSchemas(gitDirectory: File, versionConfigFile: File) {
+fun updateProviderSchemas(
+    gitDirectory: File,
+    versionConfigFile: File,
+    skipUnstableVersions: Boolean,
+    fastForwardToMostRecentVersion: Boolean,
+) {
     val schemas = Json.decodeFromString<List<SchemaMetadata>>(versionConfigFile.readText())
 
     validateIfReleaseIsPossible(schemas)
 
     val client = HttpClient(CIO) {
+        engine {
+            requestTimeout = 0
```

Comment:
What does `0` mean here? Infinite timeout? 

Even if this request is potentially really slow, it'd be nice to have a finite value here (e.g. a couple of minutes). 

Short timeout + retries might also be a nice alternative, but I'm not sure how easy to do that would be now.

Reply:
Yeah, it's infinite. Retries are a much better idea, I'll add them

Reply:
https://github.com/VirtuslabRnD/pulumi-kotlin/issues/202